### PR TITLE
Change markers: add New visual style and remove unnecessary "text semantics"

### PR DESCRIPTION
**Pull request by patrickhlauke**
In WCAG 2.1 there's one change marker for the third note in "Full pages" https://www.w3.org/TR/WCAG21/#cc2 - this currently looks odd/just like an awkward start of the sentence.

> WCAG 2.1 third note in 'Full pages', with 'New' at the start of the paragraph looking like it's just part of the sentence

In WCAG 2.2, there's a few more change markers, and these use JavaScript to add "text semantics", wrapping them in square brackets. but these don't - to my mind at least - add anything useful ... they're not read out by default by AT unless you go character by character, and as they just live in their own little paragraph, they don't really need to delimit start/end of anything and are just as understandable without the square brackets.

> ![one of WCAG 2.2's '[New]' change markers](https://user-images.githubusercontent.com/895831/96336427-9ca2ce00-1077-11eb-8ca2-f3db0f24bf22.png)

**This PR adds a more visually explicit style, and removes the square brackets**

WCAG 2.1 third note in 'Full pages', with 'New' at the start of the paragraph, now styled as bold white text on a red box with slightly rounded corners, like a tag/badge

one of WCAG 2.2's 'New' change markers, styled as bold white text on a red box with slightly rounded corners, and without the square brackets around the word

### CODE REVIEW

**Comment by mbgower** [comment]
I have added an erratum label to this, since despite the fact it is a non-normative change, it is in the TR and therefore will be recorded on the errata page.

**Comment by mbgower** [comment]
The "New" was removed from the third note of 5.2.2 Full pages in the September 2023 publish, as shown in this screenshot.

image

**Comment by mbgower** [comment]
The use of "[New]" now appears in only one location in the September 2023 version of 2.1, immediately under the heading
https://www.w3.org/TR/WCAG21/#input-modalities:

image


It appears in two contexts in the 2.2 TR:
- each of the nine new Success Criteria
- six new definitions

Note that the location of this "New" text varies between the definition and the SCs (as well as from where it appears in the 2.1 screenshot above). It immediately follows the level information for the SC:
image
For the definition, it appears underneath the definition term, where it is indented
image

**Comment by patrickhlauke** [comment]
Thanks for checking @mbgower

**Comment by mbgower** [comment]
@patrickhlauke I just wanted to confirm with you that all 3 different positions of the New 'tag' are supported by your change, and that it can handle both the single occurrence in 2.1 and the 15 additional in 2.2?

**Comment by patrickhlauke** [comment]
@mbgower yes, the changes in this PR do support them

Lone one in 2.1

screenshot of the single case in 2.1

One of the ones from 2.2

screenshot of one of the 'new' markers in 2.2

**Comment by GrahamTheDevRel** [comment]
If we want to meet AAA standards, we could change the red colour to #B60000 - this would meet AAA contrast requirements at all text sizes.

File: guidelines/guidelines.css [hunk 1]
```diff
@@ -3,7 +3,13 @@
 }
 
 .change {
-	display:inline;
+    display: inline;
+    color: #fff;
+    background: #e00;
```
Comment (patrickhlauke) [inline guidelines/guidelines.css:8]:
```suggestion
    background: #B50000;
```

**Comment by patrickhlauke** [comment]
> If we want to meet AAA standards, we could change the red colour to #B50000 - this would meet AAA contrast requirements at all text sizes.

Done.

Preview image:

Success Criterion 2.4.11 Focus Not Obscured (Minimum) (Level AA) New, with the 'New' in white on dark red

**Comment by patrickhlauke** [comment]
@alastc just checking, what's the status of this?

**Comment by iadawn** [comment]
I am slightly averse to using red to highlight something that is new. In the [design components for WAI](https://wai-website-theme.netlify.app/components/document-notes/archived/) (admittedly not TR space), red is use for a deprecated message. Yellow is used for all other messages. We don't have anything specifically for 'new' - might be worth us thinking about that.

**Comment by patrickhlauke** [comment]
Not set on the red. Could use a W3C-blue (`#005a9c`) for instance

File: guidelines/guidelines.css [hunk 2]
```diff
@@ -3,7 +3,13 @@
 }
 
 .change {
-	display:inline;
+    display: inline;
+    color: #fff;
+    background: #B50000;
+    border-radius: 0.25em;
+    padding: 0.25em;
```
Comment (patrickhlauke) [inline guidelines/guidelines.css:10]:
```suggestion
    padding: 0.25em 0.4em;
```

**Comment by iadawn** [comment]
NO (sorry, my British 'slightly averse' is really 'I don't like this at all'). Red is not to be used for marking 'new' content. Whilst it is visually salient it is used much more frequently for alerts and errors.

**Comment by patrickhlauke** [comment]
should i do a follow-up using the blue? i actually agree it might look more appropriate https://github.com/w3c/wcag/pull/1481#issuecomment-2447596547

**Comment by iadawn** [comment]
That would be great @patrickhlauke. The [WAI color palette](https://wai-website-theme.netlify.app/components/colors/) might give you some other ideas. As long as it is not red ;)

**Pull request merged**